SuperGoodSoft/solidus_taxjar
# Add transaction sync batch show page

What is the goal of this PR?
---

Show the status of individual transaction sync batches.


How do you manually test these changes? (if applicable)
---

1. Create an order with reporting turned off (so the order isnt synced immediately)
2. Go to the TaxJar settings page and click backfill transactions
3. Click the "edit" button on a batch
4. Verify you can see the status of individual orders

Merge Checklist
---


Screenshots
---
![Screen Shot 2022-06-15 at 2 51 34 PM](https://user-images.githubusercontent.com/1747836/173936863-12709aa8-42c8-4aa4-9fb3-4fe59ce38931.png)
![Screen Shot 2022-06-15 at 2 51 39 PM](https://user-images.githubusercontent.com/1747836/173936905-a9bdddfb-4033-4a31-8501-0c7a03e6f2f7.png)
